backend/canisters/community/impl/src/updates/create_channel.rs

```diff
@@ -115,7 +115,17 @@ fn create_channel_impl(args: Args, state: &mut RuntimeState) -> Response {
                 is_default: Timestamped::new(args.is_default && args.is_public, now),
```

Comment:
let mut channel = Channel { ... };

// Add the members

state.data.channels.add(channel);

Reply:
Rather than adding the channel, then immediately having to look it up again.